src/hooks/useMatchMedia.ts

```diff
@@ -16,13 +16,19 @@ export const useMatchMedia = (breakpoint: Breakpoints): boolean => {
     const [threshold, setThreshold] = useState<boolean>(
         window.matchMedia(breakpoint).matches
     );
+    const handleMatchMedia = useCallback(
+        (e: MediaQueryListEvent) => {
+            setThreshold(e.matches);
+        },
+        [threshold]
```

Comment:
threshold passed in the dependency array is not being used. is it intentional? I think an empty [] should work